### ISSUE TITLE
about the value network

### ISSUE THREAD

**Issue by daihuiao** [rvteja24/reconChessAgent#1]
thank you for your work ,i wanna know how do you train you value network ? using reinforcement learning ?

**Comment by rvteja24** [comment]
Yes that is correct, I use model free reinforcement learning for training the value network i.e. self play games are run and statistics (value and policy through MCTS. Current neural net is used to provide prior over actions and value at leaf nodes) at each timestep of the game trajectory are stored. These statistics are then used to train the network. The self play for generating data and training are alternated so that the network learns and the generated data becomes more and more efficient over time. Hope this helps.

**Comment by daihuiao** [comment]
thank you for your reply, 
there are a few questions 
1. i still can't find the code about how you train your model.what algorithm you are using?did you also train a policy network? and why didn't you use it in your MCTS to guide the search. 
2. it's seems that the value network didn't work in MCTSAgent due to stockfish is the first choice when calculate the value of the board.which means the agent don't use you neural network.
` p = self.engine.analyse(root.currentCandidate,limit=chess.engine.Limit(depth=self.depth,time=0.0000001))`
And thank you for your amazing work.
